config/locals.js

```diff
@@ -33,59 +33,122 @@ function getModelMap(configName, model) {
+ */
+function loadCampaignBotController() {
+  const campaignBotId = process.env.CAMPAIGNBOT_ID;
 
   return gambitJunior
-    .index('campaignbots')
-    .then((campaignBots) => {
-      logger.debug(`loadCampaignBots found ${campaignBots.length} campaignBots`);
+    .get('campaignbots', campaignBotId)
+    .then((campaignBot) => {
+      logger.debug(`gambitJunior found campaignBot:${campaignBotId}`);
+
       const CampaignBotController = rootRequire('api/controllers/CampaignBotController');
+      app.locals.controllers.campaignBot = new CampaignBotController(campaignBot);
+      logger.info('loaded app.locals.controllers.campaignBot');
 
-      return campaignBots.forEach((bot) => {
-        app.locals.controllers.campaignBots[bot.id] = new CampaignBotController(bot);
-        logger.info(`loaded app.locals.controllers.campaignBots[${bot.id}]`);
-      });
+      return app.locals.controllers.campaignBot;
     });
 }
 
 /**
  * Loads app.locals.campaigns from DS API.
  */
-function loadCampaigns(stringCampaignIds) {
-  logger.debug(`loadCampaigns:${stringCampaignIds}`);
+function loadCampaigns() {
   app.locals.campaigns = {};
+  app.locals.keywords = {};
+
+  const campaignIds = process.env.CAMPAIGNBOT_CAMPAIGNS;
+  logger.debug(`loadCampaigns:${campaignIds}`);
 
   return app.locals.clients.phoenix.Campaigns
-    .index({ ids: stringCampaignIds })
+    .index({ ids: campaignIds })
     .then((campaigns) => {
       logger.debug(`loadCampaigns found ${campaigns.length} campaigns`);
 
-      return campaigns.forEach((campaign) => {
-        logger.info(`loaded app.locals.campaigns[${campaign.id}]`);
-        app.locals.campaigns[campaign.id] = campaign;
-      });
+      return campaigns.map(campaign => loadCampaign(campaign));
     });
 }
 
 /**
- * Loads map of config content as app.locals.configs object instead using Mongoose find queries
- * e.g. app.locals.configs.campaign[32].title, app.locals.configs.campaignBots[41].msg_ask_why
+ * Loads app.locals.controllers.donorsChooseBot from Gambit Jr API.
  */
-function loadConfigs(uri) {
-  app.locals.configs = {};
+function loadDonorsChooseBotController() {
+  const donorsChooseBotId = process.env.DONORSCHOOSEBOT_ID;
+  logger.debug(`loadDonorsChooseBot:${donorsChooseBotId}`);
+
+  return gambitJunior
+    .get('donorschoosebots', donorsChooseBotId)
+    .then((donorsChooseBot) => {
+      logger.debug(`gambitJunior found donorsChooseBot:${donorsChooseBotId}`);
 
+      const DonorsChooseBotController = rootRequire('api/controllers/DonorsChooseBotController');
+      app.locals.controllers.donorsChooseBot = new DonorsChooseBotController(donorsChooseBot);
+      logger.info('loaded app.locals.controllers.donorsChooseBot');
+
+      return app.locals.controllers.donorsChooseBot;
+    });
+}
+
+/**
+ * Loads app.locals.db object of Mongoose models.
+ */
+function loadDb(conn) {
+  app.locals.db.campaigns = rootRequire('api/models/Campaign')(conn);
```

Comment:
Storing the Campaign model in our operations DB to allow defining keywords per environment (and [mData](https://github.com/DoSomething/gambit/wiki/Chatbot#mdata)).

The remaining collections in our `config` DB are only used by legacy endpoints -- meaning we can drop the `config` collection entirely in v3 (refs #591)